src/argilla_sdk/_api/_vectors.py

```diff
@@ -33,36 +34,31 @@ class VectorsAPI(ResourceAPI[VectorFieldModel]):
     ################
 
     @api_error_handler
-    def create(self, dataset_id: UUID, vector: VectorFieldModel) -> VectorFieldModel:
-        url = f"/api/v1/datasets/{dataset_id}/vectors-settings"
-        response = self.http_client.post(url=url, json=vector.model_dump())
-        response.raise_for_status()
-        response_json = response.json()
-        vector_model = self._model_from_json(response_json=response_json)
-        self.log(message=f"Created vector {vector_model.name} in dataset {dataset_id}")
-        return vector_model
+    def create(self, model: VectorFieldModel) -> VectorFieldModel:
+        url = f"/api/v1/datasets/{model.dataset_id}/vectors-settings"
+        response = self.http_client.post(url=url, json=model.model_dump()).raise_for_status()
+        model_created = self._model_from_json(response_json=response.json())
+        self.log(message=f"Created vector {model_created.name} in dataset {model_created.dataset_id}")
+        return model_created
```

Comment:
It's a personal preference, but I actually prefer the more verbose implementation because the stacktrace, logging, and debugging is more relevant, which is useful in a location like the API request.